tests/simple/conByteString.uplc.krun.expected

```diff
@@ -1,6 +1,6 @@
 <generatedTop>
   <k>
-    ( con bytestring #00FF ) ~> .
+    ( con bytestring #00ff ) ~> .
```

Comment:
I think you forgot to add the output from uplc for this test. In otherwords, I'm not seeing a `conByteString.uplc.expected` file. I have a fleeting question: does the grammar supported by uplc support capital letters A-F as well as a-f?

Reply:
Added.
Yes, uplc supports both small and big caps.